tachyon/zk/plonk/circuit/column_type.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020-2022 The Electric Coin Company
+// Copyright 2022 The Halo2 developers
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.halo2 and the LICENCE-APACHE.halo2
+// file.
+
+#ifndef TACHYON_ZK_PLONK_CIRCUIT_COLUMN_TYPE_H_
+#define TACHYON_ZK_PLONK_CIRCUIT_COLUMN_TYPE_H_
+
+#include <ostream>
+#include <string_view>
+
+#include "tachyon/export.h"
+
+namespace tachyon::zk {
+
+enum class ColumnType {
+  kAny,
+  kFixed,
+  kAdvice,
+  kInstance,
+};
```

Comment:
Could you add some brief explanations of each ColumnType? I think one or two lines would be enough for now.